.github/workflows/check-licenses.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        repo: [polkadot, substrate, cumulus]
+        repo: [polkadot]
```

Comment:
I am not familiar with `@paritytech/license-scanner`, will this work with `polkadot` ? Is it not needed to be `polkadot-sdk` ?

Reply:
it's already running against polkadot and succeeding so I think polkadot is right here. E.g. https://github.com/paritytech/polkadot-sdk/actions/runs/6014710075/job/16315040789?pr=1276